code/modules/atmospherics/machinery/components/binary_devices/temperature_pump.dm

```diff
@@ -0,0 +1,80 @@
+/obj/machinery/atmospherics/components/binary/temperature_pump
+	icon_state = "tpump_map-3"
+	name = "temperature pump"
+	desc = "A pump that moves heat only."
+
+	can_unwrench = TRUE
+	shift_underlay_only = FALSE
+
+	var/heat_transfer_rate = 0
+	var/max_heat_transfer_rate = 4500
```

Comment:
This comment (or lack thereof) does not meet our [documentation standards](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variabledefine), please update this.